packages/Calendar/src/Calendar/CalendarView.js

```diff
@@ -30,10 +30,19 @@ type Props = {
   setValue: number => void,
   value: ?Date,
   setView: React.Node => void,
+  type: string,
```

Comment:
If you know the types use those names:
`'multi', 'range', 'single'` and default props appropriately.